### PR TITLE
gavins push

### PR DESCRIPTION
**Pull request by wingedaunt104**
hello

### CODE REVIEW

**Comment by JFormisano** [comment]
:)

**Review by KalebLanz4 — COMMENTED** [review]
work

**Review by KalebLanz4 — APPROVED** [review]
approve

**Review by KalebLanz4 — APPROVED** [review]
approve

File: frontend/frontendJS.js [hunk 1]
```diff
@@ -19,7 +17,7 @@ function showScreen(screenId) {
 fetch(`/fighters/${fighterName}`)
     .then(response => response.json())
     .then(data => {
-    console.log(data); // Process the fighter's data (e.g., display it on the page)
```
Comment (KalebLanz4) [inline frontend/frontendJS.js:22]: this one

**Review by KalebLanz4 — APPROVED** [review]
approve

File: frontend/frontendJS.js [hunk 2]
```diff
@@ -3,10 +3,8 @@ function showScreen(screenId) {
     document.querySelectorAll('.screen, #front-page').forEach(screen => {
         screen.style.display = 'none';
     });
-
```
Comment (KalebLanz4) [inline frontend/frontendJS.js:6]: seen

File: frontend/frontendJS.js (hunk 1 above)
Comment (KalebLanz4) [inline frontend/frontendJS.js:22]: seen

**Review by KalebLanz4 — APPROVED** [review]
approve

**Review by KalebLanz4 — COMMENTED** [review]
p

**Review by KalebLanz4 — COMMENTED** [review]
p

**Pull request closed without merging**